packages/lesswrong/components/Layout.tsx

```diff
@@ -582,18 +461,7 @@ const Layout = ({currentUser, children, classes}: {
                   </ErrorBoundary>
                   {!currentRoute?.fullscreen && !currentRoute?.noFooter && <Footer />}
                 </div>
-                {isLW && standaloneNavigation && <div className={classes.imageColumn}>
-                  {/* Background image shown in the top-right corner of LW. The
-                    * loading="lazy" prevents downloading the image if the
-                    * screen-size is such that the image will be hidden by a
-                    * breakpoint. */}
-                  <CloudinaryImage2
-                    loading="lazy"
-                    className={classes.backgroundImage}
-                    publicId="ohabryka_Topographic_aquarelle_book_cover_by_Thomas_W._Schaller_f9c9dbbe-4880-4f12-8ebb-b8f0b900abc1_m4k6dy_734413"
-                    darkPublicId={"ohabryka_Topographic_aquarelle_book_cover_by_Thomas_W._Schaller_f9c9dbbe-4880-4f12-8ebb-b8f0b900abc1_m4k6dy_734413_copy_lnopmw"}
-                  />
-                </div>}
+                {isLW && <LWBackgroundImage standaloneNavigation={standaloneNavigation} />}
```

Comment:
So this is changing from being rendered only when `standaloneNavigation` is true to always rendering something, but different things depending on e.g. review phase?